frontend/src/modules/Intersection/view/components/readoutWrapper.tsx

```diff
@@ -78,24 +79,25 @@ export function ReadoutWrapper(props: ReadoutWrapperProps): React.ReactNode {
         [readoutMd, props.workbenchServices, props.viewContext, props.wellboreHeaderUuid, moduleInstanceId]
     );
 
-    const handleReadoutItemsChange = React.useCallback(function handleReadoutItemsChange(
-        event: EsvIntersectionReadoutEvent
-    ): void {
-        setReadoutItems(event.readoutItems);
-        const items = event.readoutItems;
-        const wellboreReadoutItem = items.find((item) => isWellborepathLayer(item.layer));
-        const md = wellboreReadoutItem?.md;
-        setReadoutMd(md ?? null);
-    },
-    []);
-
-    const makeLabelFromLayer = React.useCallback(
-        function makeLabelFromLayer(layer: Layer<any>): string | null {
-            return props.layerIdToNameMap[layer.id] ?? null;
+    const _formatEsvLayout = React.useCallback(
```

Comment:
Why the leading `_`? You could consider using a `function` here instead of an arrow function.

Reply:
Don't think I intended for the `_` to be there, will remove it. Regarding the method, I'm unsure; see my previous comment